scripts/oneoff/inject-framework-dates.py

```diff
@@ -0,0 +1,125 @@
+        'intentionToAwardAtUTC': '1970-01-01T00:00:00Z',
+        'frameworkLiveAtUTC': '1970-01-01T00:00:00Z',
+    },
+    'g-cloud-5': {
+        'clarificationsCloseAtUTC': '1970-01-01T00:00:00Z',
+        'clarificationsPublishAtUTC': '1970-01-01T00:00:00Z',
+        'applicationsCloseAtUTC': '1970-01-01T00:00:00Z',
+        'intentionToAwardAtUTC': '1970-01-01T00:00:00Z',
+        'frameworkLiveAtUTC': '1970-01-01T00:00:00Z',
+    },
+    'g-cloud-6': {
+        'clarificationsCloseAtUTC': '1970-01-01T00:00:00Z',
+        'clarificationsPublishAtUTC': '1970-01-01T00:00:00Z',
+        'applicationsCloseAtUTC': '1970-01-01T00:00:00Z',
+        'intentionToAwardAtUTC': '1970-01-01T00:00:00Z',
+        'frameworkLiveAtUTC': '1970-01-01T00:00:00Z',
+    },
+    # End Unix epoch time polyfill.
+
+    'g-cloud-7': {
+        'clarificationsCloseAtUTC': '2015-09-22T17:00:00Z',
+        'clarificationsPublishAtUTC': '2015-09-29T17:00:00Z',
+        'applicationsCloseAtUTC': '2015-10-06T15:00:00Z',
+        'intentionToAwardAtUTC': '2015-11-09T12:00:00Z',
+        'frameworkLiveAtUTC': '2015-11-23T12:00:00Z',
+    },
+    'g-cloud-8': {
+        'clarificationsCloseAtUTC': '2016-07-07T16:00:00Z',
+        'clarificationsPublishAtUTC': '2016-06-14T16:00:00Z',
+        'applicationsCloseAtUTC': '2016-06-23T16:00:00Z',
+        'intentionToAwardAtUTC': '2016-07-18T11:00:00Z',
+        'frameworkLiveAtUTC': '2016-08-01T11:0:00Z',
+    },
+    'g-cloud-9': {
+        'clarificationsCloseAtUTC': '2017-03-28T16:00:00Z',
+        'clarificationsPublishAtUTC': '2017-04-04T16:00:00Z',
+        'applicationsCloseAtUTC': '2017-04-11T16:00:00Z',
+        'intentionToAwardAtUTC': '2017-05-18T11:00:00Z',
+        'frameworkLiveAtUTC': '2017-05-22T11:00:00Z',
+    },
+    'g-cloud-10': {
+        'clarificationsCloseAtUTC': '2018-05-09T16:00:00Z',
+        'clarificationsPublishAtUTC': '2018-05-16T16:00:00Z',
+        'applicationsCloseAtUTC': '2018-05-23T16:00:00Z',
+        'intentionToAwardAtUTC': '2018-06-18T11:00:00Z',
+        'frameworkLiveAtUTC': '2018-07-02T11:00:00Z',
+    }
+}
+
+
+def inject_framework_dates(stage):
+    data_api_token = get_auth_token('api', stage) if stage != 'development' else 'myToken'
+    data_api_client = DataAPIClient(get_api_url('api', stage), data_api_token)
+
+    for framework_slug, framework_data in FRAMEWORKS_AND_DATES.items():
+        print(f'Injecting dates for {framework_slug}: {framework_data}')
+        try:
+            data_api_client.update_framework(framework_slug=framework_slug,
+                                             data=framework_data,
+                                             user=f'{getpass.getuser()} - '
+                                                  f'digitalmarketplace-scripts/scripts/'
+                                                  f'oneoff/inject-framework-dates.py')
+
+        except Exception as e:
+            print(f'Failed with {e}')
```

Comment:
Logging out the frameworks_slug might be useful.

Reply:
👍